libs/blocks/merch-card/merch-offer-select.js

```diff
@@ -3,12 +3,17 @@ import { decorateButtons } from '../../utils/decorate.js';
 import '../../deps/merch-offer-select.js';
 
 function createDynamicSlots(el, bodySlot) {
-  const price = createTag('h5', { class: 'merch-card-price' });
-  price.append(createTag('span', { slot: 'price', is: 'inline-price' }));
-  bodySlot.append(price);
-
+  const pricePlaceholder = el.querySelector("span[is='inline-price']");
+  if (pricePlaceholder) {
+    pricePlaceholder.parentNode.replaceChild(createTag('span', { slot: 'price', is: 'inline-price' }), pricePlaceholder);
+  } else {
+    const priceSlot = createTag('h5', { class: 'merch-card-price' });
+    createTag('span', { slot: 'price', is: 'inline-price' }, null, { parent: priceSlot });
+    bodySlot.append(priceSlot);
+  }
   const p = createTag('p', { class: 'action-area' });
-  p.append(createTag('a', { slot: 'cta', is: 'checkout-link' }));
+  createTag('a', { slot: 'secondary-cta', is: 'checkout-link' }, null, { parent: p });
```

Comment:
same here, `createTag('a', { slot: 'secondary-cta', is: 'checkout-link' }` gives the impression that you create a checkout link but it is not because it must be created via `merch` autoblock.

Reply:
we can probably remove `is: 'inline-price' ` part. and only keep ` slot: 'price'`
and as discussed offline, maybe it shouldn't be called slot at all since its not used as one.. but that can be refactored later.